lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
@markusweimer any comment on the above? One more point to consider: Configurations.Merge right now does not do dedupe. So more times we have to call Configurations.Merge, more issues we have. In case of ISet the chances will increase.